pom.xml

```diff
@@ -42,6 +42,8 @@
 
     <properties>
         <jenkins.version>1.580.1</jenkins.version>
+        <java.level>6</java.level>
+        <findbugs.failOnError>false</findbugs.failOnError>
```

Comment:
The result was:

```
FindBugs: 0 warnings from one analysis.

No warnings since build 54.
Still 79 days before reaching the previous zero warnings highscore.
```

Better. I prefer to leave the default value `<findbugs.failOnError>true</findbugs.failOnError>`